src/Application.php

```diff
@@ -70,6 +91,9 @@ public function run()
         } catch (PageNotFoundException $e) {
             $errorController = DIC::newErrorController();
             $errorController->showError404();
+        } catch (SecurityException $e) {
```

Comment:
Is each SecurityException related to CSRF? Maybe you can rename it CSRFSecurityException?